app/controllers/orders_controller.rb

```diff
@@ -18,8 +19,7 @@ def index
   def create
     @order = Order.create(user_id:     current_user.id,
                           status:      "ordered")
-    @order.create_order_items(@cart)
-    @order.update_attributes(total_price: @order.order_total)
+    session[:cart] = []
```

Comment:
at this point, I wasn't really worrying about creating the order details but just making sure we could get to this point.